lib/sidekiq/statsd/server_middleware.rb

```diff
@@ -24,7 +20,7 @@ def initialize(options = {})
                    port:           8125,
```

Comment:
Should we drop the `host` and `port` options if `statsd` is now required?

Reply:
Good catch! aeef8a3ae636b173cca3c5cbae427ea0d8e82142